sdk/communication/communication-phone-numbers/samples/v1/javascript/README.md

```diff
@@ -4,32 +4,35 @@ languages:
   - javascript
 products:
   - azure
+  - azure-communication-services
 urlFragment: communication-phone-numbers-javascript
 ---
 
-# Azure Communication Service Phone Numbers client library samples for JavaScript
+# Azure Communication Services client library samples for JavaScript
 
-These sample programs show how to use the JavaScript client library for Azure Communication Service Phone Numbers.
+These sample programs show how to use the JavaScript client libraries for Azure Communication Services in some common scenarios.
 
-| **File Name**                                                     | **Description**                                                                    |
-| ----------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
-| [purchasePhoneNumber.js][purchasephonenumber]                     | uses the PhoneNumbersClient to purchase a phone number                             |
-| [releasePhoneNumber.js][releasephonenumber]                       | uses the PhoneNumbersClient to release a purchased phone number                    |
-| [updatePhoneNumberCapabilities.js][updatephonenumbercapabilities] | uses the PhoneNumbersClient to update the capabilities of a purchased phone number |
-| [getPurchasedPhoneNumber.js][getpurchasedphonenumber]             | uses the PhoneNumbersClient to get a purchased phone number                        |
-| [getPurchasedPhoneNumbers.js][getpurchasedphonenumbers]           | uses the PhoneNumbersClient to list all purchased phone numbers                    |
+| **File Name**                                                     | **Description**                                                                                       |
+| ----------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------- |
+| [getPurchasedPhoneNumber.js][getpurchasedphonenumber]             | Demonstrates how to us the PhoneNumbersClient to get a purchased phone number.                        |
```

Comment:
typo: use instead of us

But I think it's better to not repeat "Demonstrates how to use the PhoneNumbersClient to" everywhere. Maybe we can just rewrite it to something more active and natural, like "Retrieve a previously purchased phone number.", "Purchase a phone number and use it to send an SMS."

Reply:
This is generated now and is a compilation of the summary comment at the top of each sample. I'll update those comments.